hack/make-rules/test-integration.sh

```diff
@@ -62,7 +62,7 @@ runTests() {
   make -C "${KUBE_ROOT}" test \
       WHAT="$(kube::test::find_integration_test_dirs ${2-} | paste -sd' ' -)" \
       KUBE_GOFLAGS="${KUBE_GOFLAGS:-} -tags 'integration no-docker'" \
-      KUBE_TEST_ARGS="--vmodule=garbage*collector=6" \
+      KUBE_TEST_ARGS="--vmodule=garbage*collector*=6 --alsologtostderr=true" \
```

Comment:
this should really be 
```bash
0KUBE_TEST_ARGS="${KUBE_TEST_ARGS:-} --vmodule=garbage*collector*=6 --alsologtostderr=true" \
```
rather than losing whatever was previously set in $KUBE_TEST_ARGS.